Assets/Packages/com.studio23.ss2.upmassistant/Editor/FileEditorWindowController.cs

```diff
@@ -0,0 +1,61 @@
+using UnityEngine;
+using UnityEditor;
+using System.IO;
+
+namespace Studio23.SS2.UPMAssistant.Editor
+{
+    public class FileEditorWindowController : EditorWindow
+    {
+        private static string markupText = ""; 
+        private static string filePath = "";  
+        
+        public static void ShowWindow(string path)
+        {
+            if (!string.IsNullOrEmpty(path))
+            {
+                filePath = path;
+                var window =GetWindow<FileEditorWindowController>("UPM Editor Window");
+                window.minSize = new Vector2(600, 600); 
+                markupText = File.ReadAllText(filePath);
+            }
+            else
+            {
+                Debug.LogError("File path is empty!");
+            }
+        }
+
+        private void OnGUI()
+        {
+            GUILayout.Label("UPM Editor", EditorStyles.boldLabel);
+            GUILayout.Label($"File Location: {filePath}", EditorStyles.label);
+            
+            markupText = EditorGUILayout.TextArea(markupText, GUILayout.Height(500));
```

Comment:
Depending on the use case, you might want to use EditorGUI.TextField instead of EditorGUILayout.TextArea for markupText. The choice between the two depends on whether you want a single-line or multi-line text input.